LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan/DevAddr.cs

```diff
@@ -56,6 +75,21 @@ public Span<byte> Write(Span<byte> buffer)
             return buffer[Size..];
         }
 
+        /// <summary>
+        /// Creates a <see cref="DevAddr"/> where its <see cref="NetworkId"/> is set to 0,
+        /// representing a private network address.
+        /// </summary>
+        public static DevAddr Private0(int address) => new DevAddr(0, address);
```

Comment:
Why add static members to production code which are only used from test code?

Reply:
This creates a private network address, which is perfectly valid for production too.